lifecycle/lifecycle/endpoints/auth.py

```diff
@@ -80,6 +80,12 @@ def _get_esc_token(esc_id: str, request: Request):
         auth_subject = get_auth_subject_by_esc(esc_model)
         return {'token': auth_subject.token}
 
+    @api.post('/auth/token/user/{username}/regenerate')
+    def _generate_tokens_for_user(request: Request, username: str):
```

Comment:
Since the argument `username` is not used, it can be deleted.

This might be misinterpreted by someone using this API. 
Imagine the Administrator who finds this endpoint on a Swagger page. He thinks it would be great to regenerate someone else's token, but to his surprise, he ends up with its own token being regenerated.

Reply:
Ah, I see then it collides with already existing one `/auth/token/user/regenerate`, I think it's more reasonable to rename the latter to `/auth/token/user/all/regenerate` and then we can reclaim `/auth/token/user/regenerate`.

Reply:
Yeah, I kept the naming scheme for exactly the reason you suspect. Are we good to break API like that, or should I change this pull request so it goes into some "future major version" branch rather than master?

Reply:
There's an alternative for just adding a check that a user only ever gets to call the endpoint with their own name in it.

Reply:
I'm completely fine to change the path of `_generate_tokens_for_all_users` endpoint. You can notice that it's not being used by the code. It's an administrative endpoint to be clicked by admin from a Swagger page.